setup.cfg

```diff
@@ -2,7 +2,8 @@
 description-file = README.md
 
 [tool:pytest]
-addopts = --ignore=scripts -n auto --dist=loadfile
+#addopts = --ignore=scripts -n auto --dist=loadfile
+addopts = --ignore=scripts
```

Comment:
I'm happy with making this change (we discussed it in a recent telecon) but we should update the CI calls to use xdist explicitly. Alternatively, if this was a mistake, we can make that change in another PR.

Reply:
Oh yeah this was totally a mistake. If you want I can just make the changes all here though.

Reply:
Let's do it in another (tiny) PR